src/bundle/Resources/views/content/tab/content.html.twig

```diff
@@ -1,19 +1,38 @@
-{% for group in fieldDefinitionsByGroup if group.fieldDefinitions|length > 0 %}
-    <section class="ez-fieldgroup">
-        <h5 class="ez-fieldgroup-name">{{ group.name }}</h5>
-        {% for fieldDefinition in group.fieldDefinitions %}
-            <div class="ez-content-field">
-                <p class="ez-content-field-name">{{ ez_field_name(content, fieldDefinition.identifier) }}:</p>
-                <div class="ez-content-field-value">
-                    {% if ez_is_field_empty(content, fieldDefinition.identifier) %}
-                        <em>{{'fieldview.field.empty'|trans({}, 'fieldview')|desc('This field is empty')}}</em>
-                    {% else %}
-                        {{ ez_render_field(content, fieldDefinition.identifier, {
-                            template: '@EzPlatformAdminUi/content_fields.html.twig'
-                        }) }}
-                    {% endif %}
+<section class="ez-fieldgroup ">
+    <h5 class="ez-fieldgroup-name clearfix">
```

Comment:
This HTML is invalid. According to the HTML specification, `h5` cannot have `div` inside.